### ISSUE TITLE
Could you provide the complete hyperparameter configuration in `parse_arguments.py` for the datasets.

### ISSUE THREAD

**Issue by chengpei-wu** [benfinkelshtein/CoGNN#4]
I attempted to reproduce the accuracy reported in the paper; however, I was only able to achieve ~**85%** accuracy on the Roman-Empire dataset, despite trying various parameter combinations.

Could you provide the complete hyperparameter configuration in `parse_arguments.py` for the datasets.

Thanks. @benfinkelshtein

**Comment by benfinkelshtein** [comment]
That is very unusual. Regardless, the hyper-parameter grids are specified in Tables 10-14 in the Appendix.
